plugin

```diff
@@ -89,6 +89,7 @@
   "DavidFW1960/bom-weather-card",
   "DBa2016/power-usage-card-regex",
   "dbuezas/lovelace-plotly-graph-card",
+  "dckiller51/lovelace-body-miscale-card",
```

Comment:
You can only add 1 item for each PR.

Reply:
Sorry when I added the second one it was automatically added to this PR.